src/test/java/uk/gov/hmcts/darts/transcriptions/service/impl/AdminTranscriptionServiceTest.java

```diff
@@ -353,4 +364,70 @@ private static TranscriptionSearchResult createTranscription(int seed) {
             seed % 2 == 0);
     }
 
+    @Nested
+    class ApproveDeletionTests {
+
+        @Test
+        void shouldApproveDeletionWhenDocumentExists() {
+            // Given
+            Integer documentId = 1;
+            TranscriptionDocumentEntity documentEntity = mock(TranscriptionDocumentEntity.class);
+            ObjectAdminActionEntity objectAdminActionEntity = mock(ObjectAdminActionEntity.class);
+            UserAccountEntity userAccount = mock(UserAccountEntity.class);
+            AdminApproveDeletionResponse expectedResponse = mock(AdminApproveDeletionResponse.class);
+
+            when(transcriptionDocumentRepository.findById(documentId)).thenReturn(Optional.of(documentEntity));
+            when(objectAdminActionRepository
+                     .findByTranscriptionDocument_IdAndObjectHiddenReasonIsNotNullAndObjectHiddenReason_MarkedForDeletionTrue(documentId))
+                .thenReturn(Optional.of(objectAdminActionEntity));
+            when(userIdentity.getUserAccount()).thenReturn(userAccount);
+            when(transcriptionResponseMapper.mapAdminApproveDeletionResponse(documentEntity, objectAdminActionEntity)).thenReturn(expectedResponse);
+
+            // When
+            AdminApproveDeletionResponse actualResponse = adminTranscriptionService.approveDeletionOfTranscriptionDocumentById(documentId);
+
+            // Then
+            verify(transcriptionApproveMarkForDeletionValidator).validate(documentId);
+            verify(objectAdminActionEntity).setMarkedForManualDeletion(true);
+            verify(objectAdminActionEntity).setMarkedForManualDelBy(userAccount);
+            verify(objectAdminActionEntity).setMarkedForManualDelDateTime(any(OffsetDateTime.class));
+            verify(objectAdminActionRepository).save(objectAdminActionEntityArgumentCaptor.capture());
```

Comment:
You use a Capture here but never validate the content of the capture?

Reply:
Done